### ISSUE TITLE
Still maintain?

### ISSUE THREAD

**Issue by lyseiha** [futurepress/epubjs-rn#191]

**Comment by vendeza** [comment]
Nope, but the latest stable version of @ottofeller/epubjs-rn. Only for 0.62.2 RN project version.

**Comment by ndviet1001** [comment]
For anyone need this: I created a fork for RN 0.64.4: [@ndvi3t/epubjs-rn](https://www.npmjs.com/package/@ndvi3t/epubjs-rn).
